src/main/java/io/github/apickledwalrus/skriptgui/SkriptGUI.java

```diff
@@ -22,12 +22,22 @@ public class SkriptGUI extends JavaPlugin {
 	@Override
 	public void onEnable() {
 		Plugin skript = getServer().getPluginManager().getPlugin("Skript");
-		if (skript == null || !skript.isEnabled()) {
-			getLogger().severe("Could not find Skript! Make sure you have it installed and that it properly loaded. Disabling...");
+		Version MINIMUM_VERSION_SUPPORT = new Version(2,6,2);
```

Comment:
Should use camel case
```suggestion
		Version minimumSupportedVersion = new Version(2, 6, 3);
```